tests/lib/snaps/shell/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: shell
+version: 1.0
+summary: Simple POSIX shell snap
+description: |
+    This snap contains a simple shell.
+    It is a replacement for snapd-hacker-toolbelt.
+apps:
+    shell:
+        command: ../../../bin/sh
```

Comment:
Not totally sure but maybe you could use the `test-snapd-tools.cmd` command, take a look at https://github.com/snapcore/snapd/blob/master/tests/lib/snaps/test-snapd-tools/bin/cmd